Templates/Per Tenant Extension/.github/workflows/PullRequestHandler.yaml

```diff
@@ -103,21 +98,16 @@ jobs:
       secrets: 'licenseFileUrl,insiderSasToken,keyVaultCertificateUrl,keyVaultCertificatePassword,keyVaultClientId,gitHubPackagesContext,applicationInsightsConnectionString'
       publishThisBuildArtifacts: ${{ needs.Initialization.outputs.workflowDepth > 1 }}
 
-  PostProcess:
+  StatusCheck:
     runs-on: [ windows-latest ]
     needs: [ Initialization, Build ]
     if: (!cancelled())
```

Comment:
Alternatively, the job can _always_ run and check for canceled task.

Reply:
I think it'd be better just to skip it in that case. From what I can see, in order for a GitHub status check to be "green" the job has to run and succeed. If the workflow is cancelled and the "Pull Request Status Check" task is skipped, then status check won't be green. 

Doesn't this give us the behavior we want?